### DISCUSSION TITLE
Schema.org, vocês utilizam?

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#810] [Discussão]
Pessoal, vocês acham muito importante o uso dos dados estruturados? Apesar de sempre colocar, eu nunca entendi de verdade a sua real importância. Vejo várias estruturas diferentes sendo montadas em sites parecidos e tudo mais, existe algum padrão nesses dados? Existe alguma diferença entre usar o Microdata ou JSON-LD?

**Comment by kvnol** [comment]
@heliomsolivas sempre que possível aplico o Schema nos meus projetos, dá um significado a mais para o conteúdo do seu site. Não sou especialista no assunto, mas segue alguns artigos:

- [Schema.org: Dando Significado ao conteúdo na internet](http://blog.popupdesign.com.br/schema-org-dando-significado-ao-conteudo-na-internet/)
- [Introdução ao Schema.org](https://www.devmedia.com.br/introducao-ao-schema-org/27669)

E algumas pessoas que devem manjar bem mais do que eu no assunto, a @talitapagani e o @reinaldoferraz :smile:

**Comment by JulianoPadilha** [comment]
Eu faço o curso de extensão/especialização da PUC-SP em Web Semântica e Processamento de Linguagem Natural e lá vemos profundamente a importância de dados estruturados. 

A estruturação dos dados visa dar significado e manter uma linguagem comum entre nós e a máquina, fazendo com que contextos que antes fariam sentido apenas para nós, seja levado para mecanismos de busca, mecanismos de acessibilidade, consumo de dados por meio de linguagens de serialização (XML, JSON...), etc..

Dados estruturados também permite dar um vocabulário mais profundo para sua aplicação. Têm um rapaz do curso que está trabalhando em um chatbot e para fazer com que uma "máquina" entenda contextos por meio de informações é preciso um conjunto de técnicas e ferramentas que os dados estruturados pode oferecer.

Essa é uma representação da "Pilha da Web Semântica" e que diz muito sobre o quão não trivial é o assunto.
![screen shot 2017-11-13 at 10 24 43 pm](https://user-images.githubusercontent.com/2096886/32756545-c9a21f60-c8c1-11e7-957e-76010773e94c.png)

Ainda estou iniciando nesse assunto, mas sem dúvidas é algo que devemos nos importar. Ouvimos falar muito pouco sobre dados estruturados e semântica indo além de técnicas de SEO. A web está evoluindo e como nós damos significado para o que criamos precisa evoluir junto. 

Segue a grande do curso que estou fazendo e o link para os slides das aulas:
https://newtoncalegari.com.br/post/curso-web-semantica/

Espero ter ajudado. :)

**Comment by bernardodiasc** [comment]
Vale comentar que a internet foi criada pra compartilhar conteúdo. Qualquer linguagem ou tecnologia, nesse cenário, eh secundário. Os dados e suas estruturas são a parte fundamental da coisa toda. :)

**Comment by felipefialho** [comment]
Coloquei a vida toda.

Além de tudo que falaram, tem uma integração animal com mecanismos de busca, sendo possível fazer coisas maneiras, como [adicionar rating direto na pesquisa do Google](https://www.ianbrodie.com/get-star-ratings-google-search-results/).

**Comment by talitapagani** [comment]
Eu procuro usar sempre que possível. O uso de microdata ou RDFa enriquece a semântica do conteúdo e possibilita que mecanismos de busca entendam estas informações dentro do contexto definido e facilita a outras aplicações reconhecer e importar estes dados de seu site.

O JSON-LD e a microdata são ambos voltados para web semântica e Linked Data, mas suas aplicações são diferentes. A microdata é usada dentro dos documentos HTML para criar uma estrutura de conteúdos semânticos. Escrevi há uns 7 anos um [post no Tableless sobre Microdata](https://tableless.com.br/introducao-a-microdata-no-html5/), é um pouco antigo, mas a parte teórica ainda é válida, rsrs.

O @JulianoPadilha passou um conteúdo ótimo do Newton Calegari que trabalha no W3C e manja muito do assunto. Complementando, tenho material de uma palestra que ministrei sobre web semântica no Senac que talvez possa ajudar:
https://www.slideshare.net/talitapagani/web-semntica-roadshow-ti-senac-sp

**Comment by reinaldoferraz** [comment]
Sou da linha da Talita. Enriquece o conteúdo e dá mais significado. 
O @newtoncalegari tem bastante material mesmo sobre o assunto e pode ajudar ;)
